gcs-action/.prettierignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
Do we need this ignore file? I'm not seeing prettier or eslint check those folders

Reply:
Done